go_modules/spec/fixtures/projects/module_major_version_mismatch/main.go

```diff
@@ -0,0 +1,8 @@
+package main
+
+import (
```

Comment:
I expected this to do nothing.

It does this:
```
$ go run .
go: finding module for package github.com/dependabot-fixtures/go-major-mismatch
main.go:4:2: module github.com/dependabot-fixtures/go-major-mismatch@latest found (v1.0.0), but does not contain package github.com/dependabot-fixtures/go-major-mismatch

$ GIT_PAGER=cat git diff
diff --git a/go_modules/spec/fixtures/projects/module_major_version_mismatch/go.sum b/go_modules/spec/fixtures/projects/module_major_version_mismatch/go.sum
index c6b151390..1a2928b76 100644
--- a/go_modules/spec/fixtures/projects/module_major_version_mismatch/go.sum
+++ b/go_modules/spec/fixtures/projects/module_major_version_mismatch/go.sum
@@ -1,2 +1,3 @@
 github.com/BurntSushi/toml v0.3.1/go.mod h1:xHWCNGjB5oqiDr8zfno3MHue2Ht5sIBksp03qcyfWMU=
+github.com/dependabot-fixtures/go-major-mismatch v1.0.0 h1:0C+4pi60l+Oghr+fNTG+Pg910oRtsoR2SSrUj/4lD10=
 github.com/dependabot-fixtures/go-major-mismatch v1.0.0/go.mod h1:YxTLLhA1h7WiDf80QIWyh/apZGY/SleaxLkr0ip2FU8=
```

That worries me about the validity of this test case - I expected a working project beforehand that Dependabot would avoid breaking.

Reply:
Nice spot, fixed the referenced package name